src/k8s-engine/lima.ts

```diff
@@ -731,11 +731,13 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
   protected async lima(this: Readonly<this>, ...args: string[]): Promise<void> {
     args = this.debug ? ['--debug'].concat(args) : args;
     try {
-      await childProcess.spawnFile(LimaBackend.limactl, args,
-        { env: LimaBackend.limaEnv, stdio: console });
+      const { stdout, stderr } = await childProcess.spawnFile(LimaBackend.limactl, args,
+        { env: LimaBackend.limaEnv, stdio: ['ignore', 'pipe', 'pipe'] });
+      const formatbreak = stderr || stdout ? '\n' : '';
+
+      console.log(`> limactl ${ args.join(' ') }${ formatbreak }${ stderr }${ stdout }`);
```

Comment:
Do we have to worry about ensuring a line break between `stderr` and `stdout`?

Reply:
I think so.  I put the line break here so any stderr/stdout text starts on a new line below the args.  Since console.log injects its own newline, I clear out the line break if there's no output to avoid empty lines in the log file.

Reply:
Sorry, I was unclear: this isn't asking about `formatbreak` (that looks good as-is), I'm asking if there should be a line break of some sort between `${ stderr }` and `${ stdout }`.

Reply:
Ah! - my mistake.  stderr and stdout are empty if there is no output, but they are terminated with a newline if they do have output, so they take care of it for us.